bin/test_locales.sh

```diff
@@ -23,8 +23,8 @@ echo "creating dir...."
 mkdir -p locale/xx/LC_MESSAGES
 
 echo "copying messages.pot file...."
-cp locale/templates/LC_MESSAGES/messages.pot locale/xx/LC_MESSAGES/messages.po
+cp locale/templates/LC_MESSAGES/django.pot locale/xx/LC_MESSAGES/django.po
 
 echo "translate messages.po file...."
```

Comment:
Total nit, but we should update the comments here and in the previous comment, too.